doc/sg_execution_times.rst

```diff
@@ -0,0 +1,67 @@
+
```

Comment:
@floriankozikowski delete this file and gitignore it, it's causing the failure in the CircleCI build